docs/source/other-versions.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Other versions of Apollo Server
+---
+
+## Apollo Server 4 is generally available
+
+Apollo Server 4 is the latest version of Apollo Server and is [generally available](/resources/product-launch-stages#general-availability). Apollo considers Apollo Server 4 to be ready for use in a production environment and provides official support. There is currently no planned end-of-life date.
+
+## Previous deprecated versions
+
```

Comment:
Could we also link to the docs page for discontinuing support somewhere in this article?

I feel like a note right here could feel good:

```suggestion

> [Learn more about deprecation and end-of-life.](https://www.apollographql.com/docs/resources/product-launch-stages#stages-for-discontinuing-support)

```